src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,188 @@
-              href="#/people/maria-van-brussel-1801"
-            >
-              Maria van Brussel
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carel-haverbeke-1796">Carel Haverbeke</a>
-          </td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/emma-de-milliano-1876"
-            >
-              Emma de Milliano
-            </a>
-          </td>
-          <td>f</td>
-          <td>1876</td>
-          <td>1956</td>
-          <td>Sophia van Damme</td>
-          <td>Petrus de Milliano</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/emile-haverbeke-1877">Emile Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1877</td>
-          <td>1968</td>
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-        </tr>
+        {sortedPeople.map(person => {
+          const {
+            name,
+            sex,
+            born,
+            died,
+            motherName,
+            mother,
+            fatherName,
+            father,
+          } = person;
+
+          return (
+            <tr
+              data-cy="person"
+              key={person.slug}
+              className={classNames({
+                'has-background-warning': slug === person.slug,
+              })}
+            >
+              <td>
+                <PersonLink
+                  person={person}
+                  className={classNames({
+                    'has-text-danger': sex === 'f',
+                  })}
+                >
+                  {name}
+                </PersonLink>
+              </td>
+
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+
+              <td>
+                {mother ? (
+                  <PersonLink className="has-text-danger" person={mother}>
+                    {motherName}
+                  </PersonLink>
+                ) : (
+                  motherName || '-'
+                )}
+              </td>
+
+              <td>
+                {father ? (
+                  <PersonLink person={father}>{fatherName}</PersonLink>
+                ) : (
+                  fatherName || '-'
+                )}
+              </td>
+            </tr>
+          );
+        })}
       </tbody>
     </table>
   );
```

Comment:
The rendering part of your component is a bit large and complex. Consider breaking it down into smaller, more manageable components. This would improve readability and make the code easier to maintain.